### PR TITLE
helpers

### PR DESCRIPTION
**Pull request by alexdphan**
added helpers for gpt4 / anthropic support

### CODE REVIEW

**Comment by kamath** [comment]
@alexdphan can you provide more context in the description on what this is for? where is this used?

File: pyproject.toml [hunk 1]
```diff
@@ -29,4 +29,4 @@ requires = ["setuptools>=61.0"]
 build-backend = "setuptools.build_meta"
 
 [tool.setuptools]
-packages = ["browserbase"]
+packages = ["browserbase", "browserbase.helpers"]
```
Comment (kamath) [inline pyproject.toml:32]: idt this is necessary unless you want people pip installing only the helpers

File: browserbase/helpers/__init__.py [hunk 2]
```diff
@@ -0,0 +1,4 @@
+from .gpt4 import GPT4VImage, GPT4VImageDetail
+from .anthropic import Claude3Image
+
+__all__ = ["GPT4VImage", "GPT4VImageDetail", "Claude3Image"]
```
Comment (kamath) [inline browserbase/helpers/__init__.py:4]: seems like a no-op; wouldn't a user theoretically just import `Claude3Image` directly?

File: pyproject.toml (hunk 1 above)
Comment (alexdphan) [reply pyproject.toml:32]: I'm not sure to be honest.

File: browserbase/helpers/__init__.py (hunk 2 above)
Comment (alexdphan) [reply browserbase/helpers/__init__.py:4]: image

some context, langchain was using our sdk and the imports weren't correct.

https://docs.browserbase.com/integrations/langchain/python
If u ctrl f gpt4

image

File: pyproject.toml (hunk 1 above)
Comment (kamath) [reply pyproject.toml:32]: yeah this feels unnecessary, can delete here

File: browserbase/helpers/__init__.py (hunk 2 above)
Comment (kamath) [reply browserbase/helpers/__init__.py:4]: ah i see yeah sg

Comment (kamath) [reply browserbase/helpers/__init__.py:4]: also fwiw it looks like you're importing from `browserbase.helpers.gpt4` instead of `browserbase.helpers`

Comment (alexdphan) [reply browserbase/helpers/__init__.py:4]: the problem is that the customer code was copied and pasted into the docs from both browserbase and langchain, so should we work around that?

Comment (maxbeatty) [reply browserbase/helpers/__init__.py:4]: Can we just edit their page to make it correct? https://python.langchain.com/docs/integrations/document_loaders/browserbase/#loading-images (link at bottom)

Comment (alexdphan) [reply browserbase/helpers/__init__.py:4]: CleanShot 2024-10-01 at 16 45 11@2x

still getting this when using `browserbase.helpers`

**Pull request closed without merging**